pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -460,6 +476,7 @@ class CreateVmModal extends React.Component {
             connectionName: LIBVIRT_SYSTEM_CONNECTION,
             sourceType: LOCAL_INSTALL_MEDIA_SOURCE,
             source: '',
+            osInfoList: props.osInfoList,
```

Comment:
I don't know if this should be part of state, since it's not modified anywhere, nor it doesn't represent internal state of component. And I see you put it there only because of validateParams(), so wouldn't it be better to pass this.props.osInfoList as separate parameneter of validateParams() function?
But that's a small thing and I don't insist on that.

Reply:
Done